src/citrine/informatics/processors.py

```diff
@@ -68,7 +68,7 @@ class GridProcessor(Serializable['GridProcessor'], Processor):
         properties.Integer,
         'config.grid_dimensions'
     )
-    status = properties.String('status', serializable=False)
+    status = properties.Optional(properties.String(), 'status', serializable=False)
```

Comment:
Is the None option tested anywhere?

Reply:
I bumped into it when testing the new `archive` method because the module is serialized (removing the `status` field) and then deserialized.

Having `status` be optional is the behavior of the other modules, so this was a bug. It's no longer relevant because #547 fixes the same bug.